AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -236,34 +236,34 @@
 "accountdetails.title.newaccount" = "New Account";
 "accountdetails.title.editingaccount" = "Editing Account";
 "accounttype.cloud" = "Alfresco in the Cloud";
-"accounttype.alfrescoServer" = "Alfresco Server";
+"accounttype.alfrescoServer" = "Alfresco One";
```

Comment:
@andyhealey26 We also connect to Alfresco CE repositories, which aren't "Alfresco One" per my understanding?